.forceignore

```diff
@@ -19,3 +19,6 @@ package.xml
 
 # Allowing EBikes profile for source push/pull
 !**/E-Bikes Profile.profile-meta.xml
+
+# Temporary disable Community navigation metadata
+**navigationMenus
```

Comment:
Doesn't this need to be undone?

Reply:
Yes, let me try. I was about to do that in a separate PR because of the prerelease stuff but I guess that it should work now.